src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -34,84 +34,35 @@
   <div class="card-body">
     <div class="form-wrapper">
       {{ form_errors(categoryForm) }}
-
-      {{ ps.form_group_row(categoryForm.name, {}, {
-        'label': 'Name'|trans({}, 'Admin.Global'),
-        'help': 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}'
-      }) }}
-
-      <div class="form-group row">
-        {{ ps.label_with_help(('Displayed'|trans({}, 'Admin.Global')), ('Click on "%displayed_label%" to index the category on your shop.'|trans({'%displayed_label%': 'Displayed'|trans({}, 'Admin.Global')}, 'Admin.Catalog.Help'))) }}
-        <div class="col-sm">
-          {{ form_widget(categoryForm.active) }}
-          <small class="form-text">
-            {{ 'If you want a category to appear in the menu of your shop, go to [1]Modules > Module Manager[/1] and configure your menu module.'|trans({
-              '[1]': '<a href="' ~ path('admin_module_manage') ~ '" target="_blank" rel="noopener noreferrer nofollow">',
-              '[/1]': '</a>'
-            }, 'Admin.Catalog.Help')|raw }}
-          </small>
-        </div>
-      </div>
-
+      {{ form_row(categoryForm.name) }}
```

Comment:
This is indeed better, since at least we don't really on `ps.form_group_row` anymore, but this could still be improved a bit more so that we only rely on a single `form_row(categoryForm)`
If the form contains some very specific custom parts to should be doable with a dedicated form theme